client/src/components/ProTx/components/charts/MaltreatmentTypesPlot.js

```diff
@@ -0,0 +1,329 @@
+    data,
+    geography,
+    year,
+    geoid,
+    maltreatmentTypes
+  );
+
+  const maltreatmentTypesDataObject = getMaltreatmentTypesDataObject(
+    maltreatmentTypes,
+    maltreatmentTypesList,
+    maltreatmentTypesDataValues
+  );
+
+  // Define Plotting Helper Methods.
+
+  const getBarVertTrace = (traceY, traceX, traceName, traceFillColor) => {
+    return {
+      y: [traceY],
+      x: [traceX],
+      name: traceName,
+      type: 'bar',
+      orientation: 'v',
+      marker: {
+        line: {
+          color: ['#111111'],
+          width: 1
+        },
+        color: [traceFillColor]
+      }
+    };
+  };
+
+  const getPlotDataVertBars = typesDataArray => {
+    const newPlotData = [];
+    for (let i = 0; i < typesDataArray.length; i += 1) {
+      const yData = typesDataArray[i].value;
+      const xData = typesDataArray[i].code;
+      const tName = typesDataArray[i].name;
+      const traceFillColor = plotCategoryColors[i];
+      const type = getBarVertTrace(yData, xData, tName, traceFillColor);
+      newPlotData.push(type);
+    }
+    return newPlotData;
+  };
+
+  // Assign Plot Variables.
+
+  const plotCategoryColors = [
+    '#4363d8',
+    '#911eb4',
+    '#bcf60c',
+    '#fabebe',
+    '#808000',
+    '#000075',
+    '#808080',
+    '#ffe119',
+    '#e6beff',
+    '#3cb44b',
+    '#aaffc3',
+    '#ffd8b1',
+    '#ffffff',
+    '#46f0f0',
+    '#f032e6',
+    '#008080',
+    '#000000',
+    '#e6194b',
+    '#9a6324',
+    '#fffac8',
+    '#f58231',
+    '#800000'
+  ];
+
+  const plotConfig = {
+    doubleClickDelay: 1000,
+    responsive: true,
+    displayModeBar: false,
+    modeBarButtonsToRemove: [],
+    displaylogo: false,
+    showEditInChartStudio: false
+  };
+
+  const plotLayout = {
+    autosize: true,
+    margin: { t: 40, r: 0, b: 0, l: 0, pad: 10 },
+    xaxis: {
+      automargin: true,
+      tickangle: -90,
+      title: {
+        text: 'Maltreatment Type',
+        standoff: 20
+      }
+    },
+    yaxis: {
+      automargin: true,
+      tickangle: 0,
+      title: {
+        text: 'Total',
+        standoff: 20
+      }
+    },
+    annotations: []
+  };
+
+  const plotData = getPlotDataVertBars(maltreatmentTypesDataObject);
+
+  const plotState = {
+    data: plotData,
+    layout: plotLayout,
+    config: plotConfig
+  };
+
+  // Define Element Rendering Methods.
+
+  const getMaltreatmentChartLayout = (
+    mapTypeMaltreatment,
+    geographyMaltreatment,
+    yearMaltreatment,
+    selectedGeographicFeatureMaltreatment,
+    maltreatmentTypesDataAggregateMaltreatment,
+    maltreatmentTypesListMaltreatment,
+    plotStateMaltreatment
+  ) => {
+    return (
+      <div className="maltreatment-types-plot-layout">
+        <div className="maltreatment-types-plot-header">
+          <div className="maltreatment-types-plot-info">
+            {/* <div className="maltreatment-types-plot-info-item">
+              <div className="maltreatment-types-plot-selected-map">
```

Comment:
drop?

Reply:
Keeping for now (in case they want it back in their design).

Reply:
SCratch that, dropping it anyway. =) Backed up in a  local branch.